### PR TITLE
Docker pull at Travis CI

### PR DESCRIPTION
**Pull request by beutlich**
refs #3699: Try workaround from https://travis-ci.community/t/docker-hub-you-have-reached-your-pull-rate-limit/10517/6

### CODE REVIEW

**Comment by beutlich** [comment]
@sjoelund Please advice!

File: .travis.yml [hunk 1]
```diff
@@ -11,6 +11,9 @@ before_install:
   - pip install --user futures
   - git clone --branch 5.7.28 --depth=1 https://github.com/htacg/tidy-html5.git && ( cd tidy-html5 && cmake . && make && sudo make install )
   - sudo ldconfig
+  - echo 'DOCKER_OPTS="$DOCKER_OPTS --debug --registry-mirror=https://mirror.gcr.io"'
+  - sudo service docker restart
+  - docker system info
   - docker pull openmodelica/openmodelica:v1.12.0-minimal
```
Comment (beutlich) [inline .travis.yml:17]:
```suggestion
  - docker pull mirror.gcr.io/openmodelica/openmodelica:v1.12.0-minimal
```

**Review by sjoelund — CHANGES_REQUESTED** [review]
Google doesn't mirror all docker repos. Use docker login as documented by travis instead: https://docs.travis-ci.com/user/docker/#pushing-a-docker-image-to-a-registry

File: .travis.yml [hunk 2]
```diff
@@ -11,7 +11,10 @@ before_install:
   - pip install --user futures
   - git clone --branch 5.7.28 --depth=1 https://github.com/htacg/tidy-html5.git && ( cd tidy-html5 && cmake . && make && sudo make install )
   - sudo ldconfig
-  - docker pull openmodelica/openmodelica:v1.12.0-minimal
+  - echo 'DOCKER_OPTS="$DOCKER_OPTS --debug --registry-mirror=https://mirror.gcr.io"'
+  - sudo service docker restart
+  - docker system info
+  - docker pull mirror.gcr.io/openmodelica/openmodelica:v1.12.0-minimal
```
Comment (beutlich) [inline .travis.yml:17]:
```suggestion
  - docker pull registry-1.travis-ci.com/openmodelica/openmodelica:v1.12.0-minimal
```

**Comment by beutlich** [comment]
> 
> 
> Google doesn't mirror all docker repos. Use docker login as documented by travis instead: https://docs.travis-ci.com/user/docker/#pushing-a-docker-image-to-a-registry

I do not get the docker login working.

**Comment by beutlich** [comment]
FYI: I am considering to (temporarily) remove the checks that require the openmodelica docker image. For the moparser docker image I am considering to make use of https://github.com/modelica-tools/ModelicaSyntaxChecker instead.

**Pull request closed without merging**